<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="FileNotFoundException.xml" source-language="en-US" target-language="fr-FR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5e1b479aae634bc6fe5d96e28b51a1e223a82440c.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e1b479aae634bc6fe5d96e28b51a1e223a82440c</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.IO.FileNotFoundException">
          <source>The exception that is thrown when an attempt to access a file that does not exist on disk fails.</source>
          <target state="translated">Exception levée quand une tentative d'accès à un fichier qui n'existe pas sur le disque échoue.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.FileNotFoundException">
          <source><ph id="ph1">`FileNotFoundException`</ph> uses the HRESULT COR_E_FILENOTFOUND which has the value 0x80070002.</source>
          <target state="translated"><ph id="ph1">`FileNotFoundException`</ph> utilise le HRESULT COR_E_FILENOTFOUND dont la valeur 0 x 80070002.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.FileNotFoundException">
          <source>If your code does not have <ph id="ph1">&lt;xref:System.Security.Permissions.FileIOPermissionAttribute.PathDiscovery%2A&gt;</ph> permission, the error message for this exception may only contain file or directory names instead of fully qualified paths.</source>
          <target state="translated">Si votre code n’a pas <ph id="ph1">&lt;xref:System.Security.Permissions.FileIOPermissionAttribute.PathDiscovery%2A&gt;</ph> autorisation, le message d’erreur pour cette exception peut contenir uniquement des noms de fichier ou le répertoire au lieu des chemins qualifiés complets.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:System.IO.FileNotFoundException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with optional message string and inner <ph id="ph2">&lt;see langword="Exception" /&gt;</ph> reference.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec la chaîne de message optionnelle et la référence <ph id="ph2">&lt;see langword="Exception" /&gt;</ph> interne.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with its message string set to a system-supplied message.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec sa chaîne de message définie à un message système.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "Could not find the specified file."</source>
          <target state="translated">Ce constructeur initialise la <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> propriété de la nouvelle instance à un message système qui décrit l’erreur, telles que « trouvé le fichier spécifié. »</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Ce message prend en compte la culture système en cours.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>A description of the error.</source>
          <target state="translated">Description de l'erreur.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Le contenu du <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> doit être compréhensible pour les utilisateurs.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">L'appelant de ce constructeur doit vérifier que cette chaîne a été localisée pour la culture en cours du système.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec un message d'erreur spécifié.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance using <ph id="ph2">`message`</ph>.</source>
          <target state="translated">Ce constructeur initialise la <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> propriété de l’instance à l’aide <ph id="ph2">`message`</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>An object that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Objet qui contient les données d'objet sérialisées relatives à l'exception levée.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>An object that contains contextual information about the source or destination.</source>
          <target state="translated">Objet qui contient des informations contextuelles sur la source ou la destination.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with the specified serialization and context information.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec les informations de sérialisation et de contexte spécifiées.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>A description of the error.</source>
          <target state="translated">Description de l'erreur.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Le contenu du <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> doit être compréhensible pour les utilisateurs.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">L'appelant de ce constructeur doit vérifier que cette chaîne a été localisée pour la culture en cours du système.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Exception ayant provoqué l'exception actuelle.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Si le paramètre <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> n’est pas <ph id="ph1">&lt;see langword="null" /&gt;</ph>, l’exception actuelle est levée dans un bloc <ph id="ph2">&lt;see langword="catch" /&gt;</ph> qui gère l’exception interne.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec un message d'erreur spécifié et une référence à l'exception interne ayant provoqué cette exception.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Une exception levée en conséquence directe d'une exception précédente doit contenir une référence à celle-ci dans la propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> retourne la même valeur que celle qui a été passée dans le constructeur ou retourne <ph id="ph2">`null`</ph> si la propriété <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> ne fournit pas la valeur d'exception interne au constructeur.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</source>
          <target state="translated">Le tableau suivant montre les valeurs initiales des propriétés d'une instance de <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Value</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">La référence à l'exception interne.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Chaîne du message d'erreur.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>For more information on inner exceptions, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Pour plus d’informations sur les exceptions internes, consultez <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>A description of the error.</source>
          <target state="translated">Description de l'erreur.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Le contenu du <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> doit être compréhensible pour les utilisateurs.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">L'appelant de ce constructeur doit vérifier que cette chaîne a été localisée pour la culture en cours du système.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The full name of the file with the invalid image.</source>
          <target state="translated">Nom complet du fichier comportant l’image non valide.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message, and the file name that cannot be found.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec un message d’erreur spécifié et le nom du fichier qui n’est pas trouvé.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance using <ph id="ph2">`message`</ph> and the <ph id="ph3">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> property using <ph id="ph4">`fileName`</ph>.</source>
          <target state="translated">Le constructeur initialise la <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> propriété de l’instance à l’aide <ph id="ph2">`message`</ph> et <ph id="ph3">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> à l’aide de la propriété <ph id="ph4">`fileName`</ph>.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Message d'erreur qui explique la raison de l'exception.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The full name of the file with the invalid image.</source>
          <target state="translated">Nom complet du fichier comportant l’image non valide.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Exception ayant provoqué l'exception actuelle.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Si le paramètre <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> n’est pas <ph id="ph1">&lt;see langword="null" /&gt;</ph>, l’exception actuelle est levée dans un bloc <ph id="ph2">&lt;see langword="catch" /&gt;</ph> qui gère l’exception interne.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message, the file name that cannot be found, and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> avec un message d’erreur spécifié, le nom du fichier qui n’est pas trouvé, et une référence à l’exception interne qui est la cause de cette exception.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Une exception levée en conséquence directe d'une exception précédente doit contenir une référence à celle-ci dans la propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> retourne la même valeur que celle qui a été passée dans le constructeur ou retourne <ph id="ph2">`null`</ph> si la propriété <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> ne fournit pas la valeur d'exception interne au constructeur.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</source>
          <target state="translated">Le tableau suivant montre les valeurs initiales des propriétés d'une instance de <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Value</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">La référence à l'exception interne.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Chaîne du message d'erreur.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>For more information on inner exceptions, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Pour plus d’informations sur les exceptions internes, consultez <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FileName">
          <source>Gets the name of the file that cannot be found.</source>
          <target state="translated">Obtient le nom du fichier qui est introuvable.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FileName">
          <source>The name of the file, or <ph id="ph1">&lt;see langword="null" /&gt;</ph> if no file name was passed to the constructor for this instance.</source>
          <target state="translated">Nom du fichier, ou <ph id="ph1">&lt;see langword="null" /&gt;</ph> si aucun nom de fichier n'a été passé au constructeur pour cette instance.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FileName">
          <source>This property is read-only.</source>
          <target state="translated">Cette propriété est en lecture seule.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>Gets the log file that describes why loading of an assembly failed.</source>
          <target state="translated">Obtient le fichier journal décrivant les raisons de l'échec du chargement d'un assembly.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>The errors reported by the assembly cache.</source>
          <target state="translated">Erreurs signalées par le cache d'assembly.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>The caller does not have the required permission.</source>
          <target state="translated">L'appelant n'a pas l'autorisation requise.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>to provide evidence and view and modify policy.</source>
          <target state="translated">pour fournir la preuve, d’afficher et modifier la stratégie.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>Associated enumerations: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlEvidence" /&gt;</ph>, <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlPolicy" /&gt;</ph>.</source>
          <target state="translated">Énumérations associées : <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlEvidence" /&gt;</ph>, <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlPolicy" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Objet qui contient les données d'objet sérialisées relatives à l'exception levée.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that contains contextual information about the source or destination.</source>
          <target state="translated">Objet qui contient les informations contextuelles sur la source ou la destination.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Sets the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object with the file name and additional exception information.</source>
          <target state="translated">Définit l'objet <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> à l'aide du nom de fichier et d'autres informations se rapportant à l'exception.</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>for providing serialization services.</source>
          <target state="translated">pour fournir des services de sérialisation.</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Associated enumeration:   <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> Security action: <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityAction.LinkDemand" /&gt;</ph></source>
          <target state="translated">Énumération associée : <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> action de sécurité : <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityAction.LinkDemand" /&gt;</ph></target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>requires full trust for the immediate caller.</source>
          <target state="translated">requiert une confiance totale pour l’appelant immédiat.</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This member cannot be used by partially trusted or transparent code.</source>
          <target state="translated">Ce membre ne peut pas être utilisé par le code de confiance partielle ou transparent.</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.Message">
          <source>Gets the error message that explains the reason for the exception.</source>
          <target state="translated">Obtient le message d'erreur expliquant la raison de l'exception.</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>The error message.</source>
          <target state="translated">Message d'erreur.</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>If no message was supplied to the constructor for the current exception, this property returns a system-supplied error message.</source>
          <target state="translated">Si aucun message n’a été fourni au constructeur pour l’exception actuelle, cette propriété retourne un message d’erreur fourni par le système.</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>If the <ph id="ph1">&lt;xref:System.IO.FileLoadException.FileName%2A&gt;</ph> property is not <ph id="ph2">`null`</ph>, the message includes the file name.</source>
          <target state="translated">Si le <ph id="ph1">&lt;xref:System.IO.FileLoadException.FileName%2A&gt;</ph> propriété n’est pas <ph id="ph2">`null`</ph>, le message inclut le nom de fichier.</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>This property is read-only.</source>
          <target state="translated">Cette propriété est en lecture seule.</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.ToString">
          <source>Returns the fully qualified name of this exception and possibly the error message, the name of the inner exception, and the stack trace.</source>
          <target state="translated">Retourne le nom qualifié complet de cette exception et éventuellement le message d'erreur, le nom de l'exception interne et la trace de la pile.</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.ToString">
          <source>The fully qualified name of this exception and possibly the error message, the name of the inner exception, and the stack trace.</source>
          <target state="translated">Retourne le nom qualifié complet de cette exception et éventuellement le message d’erreur, le nom de l’exception interne et l’arborescence des appels de procédure .</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>This method overrides <ph id="ph1">&lt;xref:System.Object.ToString%2A&gt;</ph>.</source>
          <target state="translated">Cette méthode se substitue à <ph id="ph1">&lt;xref:System.Object.ToString%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>The string representation returned by this method includes the name of the exception, the value of the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property, the result of calling ToString on the inner exception, the value of the <ph id="ph2">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> property, and the result of calling <ph id="ph3">&lt;xref:System.Environment.StackTrace%2A&gt;</ph>.</source>
          <target state="translated">La représentation sous forme de chaîne retournée par cette méthode inclut le nom de l’exception, la valeur de la <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> propriété, le résultat de l’appel de ToString sur l’exception interne, la valeur de la <ph id="ph2">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> propriété et le résultat de l’appel de <ph id="ph3">&lt;xref:System.Environment.StackTrace%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>If any of these members is a null reference, its value is not included in the returned string.</source>
          <target state="translated">Si un de ces membres est une référence null, sa valeur n’est pas incluse dans la chaîne retournée.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>